quad_utils/include/quad_utils/ros_utils.h

```diff
@@ -196,15 +196,16 @@ void interpRobotPlan(quad_msgs::RobotPlan msg, double t,
 quad_msgs::MultiFootState interpMultiFootPlanContinuous(
     quad_msgs::MultiFootPlanContinuous msg, double t);
 
-/**
- * @brief Interpolate data from a robot state trajectory message.
- * @param[in] msg robot state trajectory message
- * @param[in] t Time since beginning of trajectory (will return last state if
- * too large)
- * @return Robot state message
- */
-quad_msgs::RobotState interpRobotStateTraj(quad_msgs::RobotStateTrajectory msg,
-                                           double t);
+// /**
```

Comment:
remove?

Reply:
Sure. I think no one is calling this function. But I will double check with the group.

Reply:
Just checked - interpRobotState is used in inverse dynamics, but interpRobotStateTraj is no longer used (it was originally used in trajectory publisher but that has since changed). So we can remove.